src/Input/Input.js

```diff
@@ -89,6 +129,22 @@ module.exports = kind(
 	},
 
 	/**
+	* @private
+	*/
+	components: [
+		{name: 'TooltipDecorator', kind: TooltipDecorator, autoShow: false, components: [
+			{name: 'tooltip', kind: Tooltip, floating: false, position: 'right top'}
```

Comment:
Is Tooltip's position fixed like ``position: 'right top'`` ? I wonder if Tooltip's position in RTL locale is ``position: 'left top'`` or not